### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ievans**
- Fix a 404 link
- Clean up some of the README to make eliminate useless verbosity
- Add a few things like links to our youtube channel and /learn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,19 +95,17 @@ Visit [Rule examples](https://semgrep.dev/docs/writing-rules/rule-ideas/) for us
...
 
-Visit [FAQ](https://semgrep.dev/docs/faq/) to learn more. Semgrep is supported by [r2c](https://r2c.dev).
+### More
 
-### Contributing
+* [Frequently asked questions (FAQs)](https://semgrep.dev/docs/faq/)
+* [Contributing](https://semgrep.dev/docs/contributing/)
+* [Ask questions in the r2c Community Slack](https://r2c.dev/slack)
+* [CLI usage and exit codes](https://semgrep.dev/docs/cli-usage)
+* [r2c Youtube channel with Semgrep presentation videos](https://www.youtube.com/channel/UC5ahcFBorwzUTqPipFhjkWg)
```
Comment (pabloest) [inline README.md]:
```suggestion
* [r2c YouTube channel with Semgrep presentation videos](https://www.youtube.com/channel/UC5ahcFBorwzUTqPipFhjkWg)
```

File: README.md [hunk 2]
```diff
@@ -95,19 +95,17 @@ Visit [Rule examples](https://semgrep.dev/docs/writing-rules/rule-ideas/) for us
...
 
-Visit [Contributing](https://semgrep.dev/docs/contributing/) or join the [r2c Community Slack](https://r2c.dev/slack) to learn more. Semgrep is LGPL-2.1 licensed.
```
Comment (pabloest) [inline README.md:106]: Why omit the mention of licensing? Is it expected that people would go and just read the `LICENSE` file?

File: README.md [hunk 3]
```diff
@@ -95,19 +95,17 @@ Visit [Rule examples](https://semgrep.dev/docs/writing-rules/rule-ideas/) for us
 
 ### Integrations
 
-Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations.
+Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations. When integrated into CI, Semgrep will automatically only show findings that are being introduced in the current commit.
```
Comment (dlukeomalley) [inline README.md]:
```suggestion
Visit [Integrations](https://semgrep.dev/docs/integrations/) to learn about Semgrep editor, commit, and CI integrations. When integrated into CI and configured to scan pull requests, Semgrep will only report issues introduced by that pull request; this lets you start using Semgrep without fixing or ignoring pre-existing issues!
```

File: README.md (hunk 2 above)
Comment (ievans) [reply README.md:106]: My thought was that it's redundant, but on reflection I suppose it doesn't hurt to have it in the readme either!

**Review by pabloest — APPROVED** [review]

**Pull request merged**